support/jsdoc/theme/static/styles/jsdoc-default.css

```diff
@@ -127,12 +143,32 @@ section, h1 {
 
 .nav.navbar-right .navbar-form {
     padding: 0;
-    margin: 0;
+    margin: 6px 0px;
+}
+
+.navbar-collapse.collapse {
+  display: block!important;
+}
+
+.navbar-nav>li, .navbar-nav {
+  float: left !important;
+}
+
+.navbar-nav.navbar-right:last-child {
+  margin-right: -15px !important;
+}
+
+.navbar-right {
+  float: right!important;
```

Comment:
Ah, i see.  It might not be required, since this stylesheet is loaded after bootstrap.

I'm also considering ditching Bootstrap and just doing the CSS ourselves.  We're fighting it in a couple places, and all we use it for is this navbar.